tests/projects/package-stack/stack.yaml

```diff
@@ -0,0 +1,66 @@
+# This file was automatically generated by 'stack init'
```

Comment:
this seems like a lonely `stack.yaml` does it not have any friends, e.g. source files?